src/Invitation.ts

```diff
@@ -409,4 +444,45 @@ export class SealedInvitation implements invitation.SealedInvitation {
       v2: undefined,
     })
   }
+
+  /**
+   * Create a SealedInvitation with a SealedInvitationV1 payload
+   * Will encrypt all contents and validate inputs
+   *
+   * @param sender MUST be a send key corresponding to the current viewer
```

Comment:
looks like the docstring needs to be updated with parameters per the lint warnings